www/nginx.conf

```diff
@@ -1,48 +1,37 @@
 server {
-    listen       80;
-    server_name  localhost;
-    #charset koi8-r;
-    #access_log  /var/log/nginx/host.access.log  main;
+    # listen on port 80
+    listen 8080;
```

Comment:
Should it be `listen 80;`?

Reply:
80 is a privileged port and requires root (same for 443). Since this uses the non-privileged NGINX it listens on port 8080.

Reply:
So it is just the comment

Reply:
yep